helm_deploy/whereabouts-api/templates/_envs.tpl

```diff
@@ -63,4 +63,41 @@ env:
       secretKeyRef:
         name: dps-rds-instance-output
         key: rds_instance_endpoint
+
+    - name: SQS_AWS_ACCESS_KEY_ID
+      valueFrom:
+        secretKeyRef:
+          name: ocn-events-sqs-instance-output
```

Comment:
these all look wrong - you're grabbing the offender case note event output?  Should be grabbing the whereabouts output and configuring it in cloud platform environments

Reply:
This follows offender-case-notes and there doesn't seem to be any references to sqs modules in the cloud-platform-project.

Reply:
https://github.com/ministryofjustice/cloud-platform-environments/tree/master/namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-events-dev/resources